### ISSUE TITLE
Issues with getting :root selector to work

### ISSUE THREAD

**Issue by anselmh** [styled-components/styled-components#60]
Hey guys,

really like this implementation, makes a lot of sense to me when you need this approach of styling a JS-app. However I failed to get the [`:root { }` selector](https://www.w3.org/TR/selectors/#root-pseudo) working [in a simple test](http://www.webpackbin.com/NkIaHruAZ).

The goal is to use CSS Custom Properties in an app which are declared in the `:root` element (this equals the html element but the selector has differences in specificity).

In plain CSS, [the demo can be seen here](http://jsbin.com/weqelaruyo/edit?html,css,output), if you have [a CSS Custom Property supporting browser](http://caniuse.com/#search=custom%20properties).

Any hint would be appreciated but if it’s not possible to use that at the moment, no worries — then take this report as a “I’d love to see support for this at some point in future”. :)

**Comment by mxstbr** [comment]
> Any hint would be appreciated but if it’s not possible to use that at the moment, no worries — then take this report as a “I’d love to see support for this at some point in future”. :)

This should definitely be possible, if it isn't that's a bug! Will take a look tomorrow.

**Comment by giuseppeg** [comment]
@anselmh you are importing `injectGlobal` in the wrong way.

This should work:

```javascript
import { injectGlobal } from 'styled-components';
```

**Comment by anselmh** [comment]
Ah, missed this! Thanks, that seems to work. :)

**Comment by mxstbr** [comment]
Ah, thank you @giuseppeg!
